mod_reforged/hooks/skills/special/double_grip.nut

```diff
@@ -3,6 +3,7 @@
 	// because we go through weapon types alphabetically and choose the first applicable type
 	q.m.CurrWeaponType <- null;
 	q.m.MeleeDamageMult_Dagger <- 1.2;
+	q.m.IsFreeSwapSpent <- true;
```

Comment:
The default value should be `false` to emphasize. `false` is what this variable has most of the time and which is the normal case. `true` is the temporary exception